scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -31,7 +31,8 @@
 TEST_SUITES_NOT_RUN_ON_TRAVIS = [
```

Comment:
We might have to check the e2e test suites covered in circleCI, I'll do that in my PR!